src/components/List/list.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Styles from "./list.module.css";
+
+type TodoItem = {
+  id: string;
+  title: string;
+};
```

Comment:
Even though this is a small project, you could still make use of separation of concerns and create a `types` folder inside `src` and add your types there